### PR TITLE
added hireart to showcase

### PR DESCRIPTION
**Pull request by tomtang539**
Added HireArt to the showcase

### CODE REVIEW

**Comment by mkonicek** [comment]
@tomtang539 please add this to the list of apps below, not the featured ones. Otherwise we'd have to have a "featured featured" list if all apps are featured.

See facebook.github.io/react-native/showcase.html

Can you please sign the CLA (see the comment by the bot above)?

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -209,6 +209,12 @@ var featured = [
 
 var apps = [
   {
+    name: 'HireArt',
```
Comment (mkonicek) [inline website/src/react-native/showcase.js]: Please add the app under H (the list is ordered alphabetically).

**Comment by facebook-github-bot-3** [comment]
@brentvatne would you mind taking a look at this pull request? It's been a while since the last commit was reviewed.

**Comment by mkonicek** [comment]
@tomtang539 Sorry for the delay, can you keep the list of apps sorted alphabetically please?

**Comment by mkonicek** [comment]
@tomtang539 I'll close this since there's been no reply for a while. If you want free to send a new PR and we'll review it.

**Pull request closed without merging**